src/main/java/org/jenkinsci/test/acceptance/plugins/ssh_slaves/SshSlaveLauncher.java

```diff
@@ -36,14 +36,11 @@ public SshCredentialDialog addCredential() {
 
         find(by.button("Add")).click();
 
-        if (getElement(by.xpath("//span[contains(@class,'credentials-add')]")) == null) {
+        String providerXpathExpr = "//div[contains(@class,'credentials-add-menu-items')]"
```

Comment:
this could probably be cleaned up given we are supporting 2.346 LTS and weeklies, but as this is blocking important fixes I am leaving it as is for now.